strangerstudios/paid-memberships-pro
# Showing admin notice on Delete User Form if any users for deletion have membership.

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds a notice when admin tries to delete users. It checks if any users for deletion have membership and shows an appropriate message for a single or multiple user deletion informing admin that membership and active subscriptions will be terminated after deletion.

### How to test the changes in this Pull Request:

1. Go to Users admin page.
2. Select a single user with no level for deletion. No message should be shown. 
3. Select a single user with a level for deletion. A message should be shown. 
4. Select multiple users, some with a level, for deletion. A different message should be shown. 

### Other information:


### Changelog entry
* ENHANCEMENT: Showing a notice on the Delete User form so admin knows that membership and subscriptions will be cancelled.
